File: Python/백준/Bronze/2581. 소수/소수.py
import sys
import math

M = int(sys.stdin.readline())
N = int(sys.stdin.readline())

primes = []
for num in range(M, N+1):
    if num < 2:
        continue
    is_prime = True
    for i in range(2, int(math.isqrt(num)) + 1):
        if num % i == 0:
            is_prime = False
            break
    if is_prime:
        primes.append(num)

if len(primes) == 0:
    print(-1)
else:
    print(sum(primes))
    print(primes[0])

File: C99/백준/Bronze/2581. 소수/소수.c
#include <stdio.h>
#include <math.h>

int isPrime(int n) {
    if (n < 2) return 0;
    for (int i = 2; i <= sqrt(n); i++)
        if (n % i == 0) return 0;
    return 1;
}

int main() {
    int M, N, sum = 0, min = -1;
    scanf("%d %d", &M, &N);
    for (int i = M; i <= N; i++) {
        if (isPrime(i)) {
            sum += i;
            if (min == -1) min = i;
        }
    }
    if (min == -1) printf("-1\n");
    else {
        printf("%d\n%d\n", sum, min);
    }
    return 0;
}
